2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+
+
+## Analyse existierender Geschäftsmodelle
+
+Wikipedia listet folgende [Geschäftsmodelle](https://en.wikipedia.org/wiki/Business_models_for_open-source_software)
+für "Open-Source-Software" auf:
+
+    3.1 Dual-licensing
+    3.2 Selling professional services
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.5 Selling software as a service
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+    3.12 Selling of optional proprietary extensions
+    3.13 Selling of required proprietary parts of a software product
+    3.14 Selling of proprietary update systems
+    3.15 Re-licensing under a proprietary license
+    3.16 Obfuscation of source code
+    3.17 Delayed open-sourcing
+    3.18 Open sourcing on end-of-life
+
+
+ Für die Projekte mit der Zielgruppe der Endbenutzer\*innen kommen bestimmte Modelle
+ nicht in Frage bzw. sind nicht wünschenswert (Unterschied zwischen Open-Source und freier Software).
+ Im Folgenden sind Ausschlussgründe und die davon betroffenen Geschäftsmodelle aufgelistet.
+
+
+- Nicht sinnvoll, Weil kein Weiterverkauf angestrebt wird
+    - Dual-licensing
+    - Re-licensing under a proprietary license
+- Zielkonflikt: Perfekte Software braucht keinen Support/Schulung
+    - Selling professional services
+- Problem der Datensouveränität.
+    - Selling software as a service
+
+
+- Widerspricht dem Ziel maximalen FLOSS-Anteils
+    - Selling of optional proprietary extensions
+    - Selling of required proprietary parts of a software product
+    - Selling of proprietary update systems
+    - Obfuscation of source code
+    - (Delayed open-sourcing)
+
+- Nicht nachhaltig
+    - Open sourcing on end-of-life
+
+Es bleiben also folgende Modelle übrig:
+
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+
+## Ideenskizze zur Kombination (einiger) dieser Methoden
+
+### Vorbemerkung
+"Voluntary donations" klingt, als wäre die Zahlung "gegen die ökonomische Vernunft" und allein
+aus irrationalem Altruismus oder Mitleid. Ohne tiefer in die Kritik des Modells des Homo-Oeconomicus
+einzusteigen, zwei Anmerkungen:
+
+- In vielen anderen Bereichen sind Menschen aber bereit, erhebliche Mittel "entgegen der ökonomischen Vernunft" auszugeben, Stichwort: Statuskonsum.
+- Wie oben schon erwähnt, kann die finanzielle Unterstützung eines Projektes von dem man individuell profitiert auch mit informiertem Eigeninteresse begründet werden.
+
+
+### Kombinationsmodell
+Es gibt eine Organisation (Arbeitstitel: "Funding Freedom Society" (FFS)).
```

Comment:
cf. z.B. Liberapay, Buy Me A Coffee, Patreon. Alles nicht neu, aber alles mit steuerlichem Aufwand verbunden. Während es an sich vermutlich ganz nett ist als Start-/Nebenbeifinanzierung, sind die steuerlichen Aspekte da eine Hürde.

Und Marketing. Marketing ist erforderlich, damit das überhaupt funktioniert. Und als FLOSS-Entwickler habe ich keine Lust mich mit Marketing zu beschäftigen, weil das Zeit ist die ich nicht in das Projekt investieren kann. Wenn ich sicher sein könnte, dass das Marketing einen Sinn hat…

Reply:
Das Problem mit der Regulierung/Handhabung ist ein Nebenschauplatz der durch andere Konstrukte gelöst werden könnte. s.a. https://wiki.c3d2.de/Hacker_e.G. – aber sonst ist es richtig, dass bestimmte Professionen aus Liebhaberei entstehend einen enormen Overhead bei der Kommerzialisierung erzeugen bzw. für sich genommen nicht rentabel sind.

Reply:
@horazont Wobei die steuerlichen Aspekte eigentlich überschaubarer sind als sie vielleicht im ersten Moment (wenn man das noch nie gemacht hat) erscheinen.
Sich einmal als Freiberufer beim Finanzamt zu melden und dann einmal im Jahr eine Zahl in ein zusätzliches Formular einzutragen ist weniger scary als es scheint.
Könnte man vielleicht auch mal irgendwo einen Vortrag (mit einem riesen INAL disclaimer) drüber halten - oder irgendwo jemanden finden der da tatsächlich Ahnung von an.

Reply:
s.a. Schenkung f. Zahlungen ohne Gegenleistung, sofern kein Gewerbe vorliegt.

Reply:
INAL; Die Gegenleistung die Software. Diese Art von Spenden sind idr keine Schenkungen.
Bei über 17k im Jahr fallen dann sogar noch 19% Umsatzsteuer an solange man nicht gemeinnützig ist was Open Source Software idr nicht ist. (Auch so ne Sache über die man mal mit seinem Abgeordneten reden könnte - aber das ist ein anderes Thema)
Über UmSt würde ich mir am Anfang aber mal keine Gedanken machen. Auf die 17k muss man erst mal kommen.

Reply:
Steuern: Ohne mich im Detail auszukennen, sehe ich das so wie @iNPUTmice: Lösbares Problem. Um es zu erleichtern könnte die FFS ein *HowToSteuern* erstellen, auf die wichtigsten Fälle und Regionen bezogen. In Deutschland meines Wissens einen Freibetrag von einigen Tausend Euro Gewinn für freiberufliche Tätigkeiten. Und, wenn man das Thema komplett vermeiden will: man muss sich das Geld ja nicht auszahlen lassen, sondern kann es an andere Projekte weiterleiten und damit ggf. diese positiv beeinflussen.

Fakt ist aber: das Detail-Konzept müsste mit Jura- und BW-Fachkenntnis erstellt werden. Steht schon unter nächste Schritte.

Zum Marketing. Dass es FLOSS an gutem Marketing fehlt (und die wenigsten darauf Bock haben) sehe ich auch so. Deswegen ja die Überlegung: dann muss man halt Leute dafür bezahlen. Ich könnte mir vorstellen, dass es Agenturen gibt, die sich auf sowas spezialisieren.

Das sehe ich also nicht als Gegenargument für das Konzept.

Reply:
Sich das Geld nicht auszahlen zu lassen ist steht halt ein bisschen dem Ziel entgegen, mehr Ressourcen für FLOSS-Entwicklung freizuhaben.

Reply:
Wenn das so ist, könnten Zuwendungen als Spende an eine entsprechend steuerbefreite Organisation (Stiftung, gemeinnützige Unternehmen/Verein) fließen – und zwar direkt weil sonst zwischendurch Steuern anfallen.

Reply:
@horazont: jein. Man könnte argumentieren, dass ich somit eigenes Geld zur Unterstützung der Projekte die ich wichtig finde einspare. Ich kann mir so durch eigene Leistung in meinem Projekt Gestaltungseinfluss auf andere Projekte erarbeiten (z.B. Wunsch-Feature pushen). Oder (Meta-Ebene) mir Statuskonsum erspare, weil ich meine Anerkennung durch öffentlich sichtbare Förderung anderer Projekte bekomme.

Reply:
Ich würde ja mit dem Entwickeln von Freier Software gerne erstmal bei einem Punkt ankommen, wo es mir um Statuskonsum geht. Ich glaube wir sind hier auf unterschiedlichen Ebenen unterwegs :).

(Außerdem bin ich mir nicht sicher ob das nicht trotzdem steuerlich und geldwäscherechtlich ein sehr interessantes Problem ist wenn man als Entität B quasi automatisiert Geld von A nach C/D/E schiebt…)